Tasks/ANTPreview/ant.ps1

```diff
@@ -23,12 +22,15 @@ Write-Verbose "testResultsFiles = $testResultsFiles"
 Write-Verbose "jdkVersion = $jdkVersion"
 Write-Verbose "jdkArchitecture = $jdkArchitecture"
 
-Write-Verbose "publishCodeCoverageResults = $publishCodeCoverageResults"
-Write-Verbose "codeCoverageTool = $codeCoverageTool"
-Write-Verbose "summaryFileLocation = $summaryFileLocation"
-Write-Verbose "reportDirectory = $reportDirectory"
-Write-Verbose "additionalCodeCoverageFiles = $additionalCodeCoverageFiles"
-
+$isCoverageEnabled = !$codeCoverageTool.equals("NoCoverage")
+if($isCoverageEnabled)
+{
+    Write-Verbose "codeCoverageTool = $codeCoverageTool"
+    Write-Verbose "classFilesDirectory = $classFilesDirectory"
+    Write-Verbose "classFilter = $classFilter"
+    Write-Verbose "srcDirectory = $srcDirectory"
+}
+	
 #Verify Ant build file is specified
 if(!$antBuildFile)
```

Comment:
We throw an exception if this null and dont reach to a place where we are checking for buildpath